### ISSUE TITLE
BlackWidow V3 mini hyperspeed

### ISSUE THREAD

**Issue by Dingskeheezeg** [1kc/razer-macos#409]
can you please add support for BlackWidow V3 mini hyperspeed wireless keyboard, the divice id for the dongle is 0x0271, the wired usb id is 0x0258

thx

**Comment by Dingskeheezeg** [comment]
and the bt id is 0x0272

**Comment by Dingskeheezeg** [comment]
Would it be possible to also add some more advanced lithning effects that synapse 3 provide ?
Like maybe the starlight or fire , or maybe an customized wave effect ?

thx in advance ( it would make me consider to be happy to sponsor some more $ )

**Comment by Dingskeheezeg** [comment]
what is the progress ? if would be the same level of support as the current huntsman elite that woulkd be awesome

**Comment by 1kc** [comment]
Let's track the support progress [here](https://github.com/openrazer/openrazer/issues/1544). Make sure you give openrazer your support as well. :)

**Comment by Dingskeheezeg** [comment]
i already did so :)

i would realy like to see an custoim wave effect like the cycle wher you set an number of colors in order to be shown on the wave effect on any device whoi can support it , also maybe the firefly v2 mouse pad , i have two of them.

i would be happy to donate more $ to this wonderfull project and also the open razer project ;P

**Comment by Dingskeheezeg** [comment]
i also posted for older razer divices an easy fix for synapse 2 to detect (ttested in big sur 11.4) https://insider.razer.com/index.php?threads/synapse-2-mac-fix-detecting-devices.76548/

**Comment by 2001ptrey** [comment]
Hey guys, just wanted to share that on page 10 of the [manual](http://drivers.razersupport.com//index.php?_m=downloads&_a=downloadfile&downloaditemid=4377) there is a list of keybinds which allow you to adjust the color effects of the keyboard, I just tested this and it does work on macOS. It's not perfect, but works until support gets added.

**Comment by Dingskeheezeg** [comment]
> Hey guys, just wanted to share that on page 10 of the [manual](http://drivers.razersupport.com//index.php?_m=downloads&_a=downloadfile&downloaditemid=4377) there is a list of keybinds which allow you to adjust the color effects of the keyboard, I just tested this and it does work on macOS. It's not perfect, but works until support gets added.

yes that does work even without any driver , you need to press fn + ctrl + the number of the abover row without shift, works even when not connected to any device

grts

**Comment by Qix-** [comment]
IssueHunt is abandoned. If you are putting money into it, you are losing it. Nobody will be able to receive that money as IssueHunt is not responding to withdrawal requests.

Just a heads up. It's best to just forget about it.

**Comment by MagnusStrom** [comment]
cringe.

**Comment by Qix-** [comment]
What's cringe about preventing people from losing more money?

**Comment by Dingskeheezeg** [comment]
so this means the sponsored money is not only lost, but seems also nothing happens to add this device ...

**Comment by MagnusStrom** [comment]
> What's cringe about preventing people from losing more money?

no its cringe that issuehunt isint working

**Comment by Dingskeheezeg** [comment]
deviced still not supported ... this is disappointing ...

**Comment by Dingskeheezeg** [comment]
the device is added in openrazer ...

**Comment by cron-dmn** [comment]
Any update on this ?
I still dont see my device on 0.4.6.

**Comment by MagnusStrom** [comment]
Hey guys...
This thread is depressing, but there's hope.
OpenRGB recently added support for this device, and it has macos support
[**https://openrgb.org/**](https://openrgb.org/)

**Comment by Ilyahas** [comment]
I'm also looking forward when it will be added
